plugins/set_value/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Removed
 
 ### Fixed
+* Fix wrong function from abs to fabsf
+* Use also fabs for double
```

Comment:
Not sure this will be clear to users. The changelog should be about the user-facing result, not the way it was solved

Reply:
Also, an issue would make it possible to point interested users to a more in-depth description and place for discussion

Reply:
```suggestion
* Correct some tests that would incorrectly compare floats with abs instead of
  fabs, rounding down to zero and hence never failing, even in the case of error
```